commands/FBPrintCommands.py

```diff
@@ -424,3 +425,60 @@ def run(self, arguments, options):
       actionsDescription = fb.evaluateExpressionValue('(id)[{actions} componentsJoinedByString:@", "]'.format(actions=actions)).GetObjectDescription()
 
       print '{target}: {actions}'.format(target=targetDescription, actions=actionsDescription)
+
+class FBPrintAsCurl(fb.FBCommand):
+  def name(self):
+    return 'pcurl'
+
+  def description(self):
+    return 'Print the NSURLRequest (HTTP) as curl command.'
+
+  def options(self):
+    return [
+      fb.FBCommandArgument(short='-p', long='--portable', arg='portable', boolean=True, default=False, help='Embed request data as base64.'),
```

Comment:
I don't get how 'portable' makes sense here still. It sort of makes sense, but not intuitive to me.

Since you're copy-pasting the curl command anyway, I don't see why it matters that the base64 is ugly.